### ISSUE TITLE
Decide final home for packages

### ISSUE THREAD

**Issue by wcornwell** [mwpennell/phyndr-ms#28]
@mwpennell is worried that all are tools are a bit scattered in github world

**Comment by wcornwell** [comment]
@richfitz talked to Dan about https://github.com/traitecoevo .  would it make sense to make this bigger and have a front page that organizes all of our various tools and packages?

**Comment by wcornwell** [comment]
talked to @dfalster about this.  it might be possible to have all tools and packages at https://github.com/traitecoevo with a front page that organizes them and directs people to the right tool?  Any thoughts @richfitz ?  Seems like it would be useful to have a tool directory somewhere on the www.

**Comment by richfitz** [comment]
I agree that things are scattered.  Happy to host the domain-specific ones at traitecoevo, or we could go with Cornwell lab.

FWIW, the long-term aim is to migrate some of the traitecoevo tools from there onto ropensci (the infrastructure tools, rather than the science bits) so not sure how that plays out.

**Comment by wcornwell** [comment]
I vote for traitecoevo for all the ecoevo tools, and ropensci for the infrastructure tools.  I'm willing to help build a little table of contents for the home page...

**Comment by mwpennell** [comment]
I am on board with this as well.

**Comment by mwpennell** [comment]
hey @wcornwell and @richfitz I edited the ms to have all repos (phyndr, taxonlookup, phyndr-ms) be located at traitecoevo https://github.com/mwpennell/phyndr-ms/commit/aa814e53321430a82d8da705be8764dc7a531291 

so assuming this is cool, we need to:
1. all transfer ownership of our respective repos
2. update https://github.com/mwpennell/phyndr-ms/commit/aa814e53321430a82d8da705be8764dc7a531291 to reflect the changes

**Comment by richfitz** [comment]
I've pinged @dfalster and will see if he's OK with this.  If so, then it's just an extremely small job to transfer ownership.

**Comment by dfalster** [comment]
I'm very happy for material to be hosted at traitecoevo!

**Comment by wcornwell** [comment]
Maybe eventually build a tool/dataset navigation menu from one of these http://designscrazed.org/best-free-navigation-menus/ ...

**Comment by wcornwell** [comment]
taxonlookup transfered...

**Comment by dfalster** [comment]
Cool, I’ve given team admin privileges on that repo
